.github/workflows/build_and_test.yml

```diff
@@ -80,6 +84,19 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
 
+      - name: Set up Python
+        run: |
+          if [ "${{ matrix.image }}" = "alpine317" ]; then
+            apk add python3 py3-virtualenv py3-virtualenv python3-dev gcc musl-dev linux-headers
+          elif [ "${{ matrix.image }}" = "rhel94" ]; then
+            yum install -y python3 python3-virtualenv python3-devel
+          elif [ "${{ matrix.image }}" = "fedora39" ]; then
+            dnf install -y python3 python3-virtualenv python3-devel
+          else
+            apt-get update
+            apt-get install -y python3 python3-venv python3-dev
+          fi
```

Comment:
this should be done in the container

Reply:
ya, we can do that as well, @seladb can you update the containers?

Reply:
I can do it if you want, but you just need to open a MR here:
https://github.com/seladb/PcapPlusPlus-DockerImages

Reply:
cool, I didn't know this.

Reply:
@seladb  shouldn't [/PcapPlusPlus-DockerImages](https://github.com/seladb/PcapPlusPlus-DockerImages) under the organization?

Reply:
I guess we can move it, but maybe do it after the release?

Reply:
got it.

Reply:
fixed.